static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -0,0 +1,297 @@
+import {browserHistory} from 'react-router';
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import qs from 'qs';
+
+import _EventsRequest from 'sentry/components/charts/eventsRequest';
+import DatePageFilter from 'sentry/components/datePageFilter';
+import {COL_WIDTH_UNDEFINED} from 'sentry/components/gridEditable';
+import * as Layout from 'sentry/components/layouts/thirds';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {NewQuery} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import {useQuery} from 'sentry/utils/queryClient';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import withApi from 'sentry/utils/withApi';
+import FacetBreakdownBar from 'sentry/views/starfish/components/breakdownBar';
+import Chart from 'sentry/views/starfish/components/chart';
+import EndpointTable from 'sentry/views/starfish/modules/APIModule/endpointTable';
+import DatabaseTableView from 'sentry/views/starfish/modules/databaseModule/databaseTableView';
+import {useQueryMainTable} from 'sentry/views/starfish/modules/databaseModule/queries';
+import {HOST} from 'sentry/views/starfish/utils/constants';
+import {getModuleBreakdown} from 'sentry/views/starfish/views/webServiceView/queries';
+
+const EventsRequest = withApi(_EventsRequest);
+
+const HTTP_SPAN_COLUMN_ORDER = [
+  {
+    key: 'description',
+    name: 'URL',
+    width: 400,
+  },
+  {
+    key: 'throughput',
+    name: 'Throughput',
+    width: 350,
+  },
+  {
+    key: 'p50(exclusive_time)',
+    name: 'p50',
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'transaction_count',
+    name: 'Transactions',
+    width: COL_WIDTH_UNDEFINED,
+  },
+
+  {
+    key: 'total_exclusive_time',
+    name: 'Total Time',
+    width: COL_WIDTH_UNDEFINED,
+  },
+];
+
+const DATABASE_SPAN_COLUMN_ORDER = [
+  {
+    key: 'description',
+    name: 'Query',
+    width: 400,
+  },
+  {
+    key: 'domain',
+    name: 'Table',
+    width: 100,
+  },
+  {
+    key: 'epm',
+    name: 'Tpm',
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'p75',
+    name: 'p75',
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: 'total_time',
+    name: 'Total Time',
+    width: COL_WIDTH_UNDEFINED,
+  },
+];
+
+export default function EndpointOverview() {
+  const location = useLocation();
+  const organization = useOrganization();
+  const theme = useTheme();
+
+  const {endpoint: transaction, method, statsPeriod} = location.query;
+  const pageFilter = usePageFilters();
+
+  const {
+    isLoading: isTableDataLoading,
+    data: tableData,
+    isRefetching: isTableRefetching,
+  } = useQueryMainTable({action: 'ALL', table: 'ALL'});
```

Comment:
```suggestion
  } = useQueryMainTable({});
```
Small nit, we now treat action = 'ALL' and action = undefined as the same thing, same with table. so u can omit these params.

Reply:
Ah nice, even better!